netlify.toml

```diff
@@ -1,7 +1,3 @@
-[build]
-  publish = "exampleSite/public"
-  command = "cd exampleSite && hugo --gc --themesDir ../.."
-
```

Comment:
This change needs to be reverted. I need the entire `[build]` section to remain, so that my theme builds correctly in its Netlify demo.

Reply:
Reverted in the commit 8115a28